.pre-commit-config.yaml

```
@@ -0,0 +1,114 @@
+exclude: Pipfile\.lock|migrations|\.idea|node_modules
```

There might be more files than this to ignore as well, but I think we need at least these :thinking: 

```suggestion
exclude: pnpm-lock\.yaml|\.idea|node_modules|nuxt-template-overrides|\.storybook
```

---

@sarayourfriend I added an exception for `nuxt-template-overrides` in ESLint. Even without excluding the others, all checks are passing. So I think it's better to leave them in.